Gauss spellings and more

banesullivan opened the pull request:
Gauss should be capitalized as it's referring to a human (proper noun).

Note: what I did doesn't seem to work. Is the dictionary case insensitive?

Output from a sample project:

```
./section5.tex:102: gauss'  ==> gauss'
./section1.tex:669: Gauss'  ==> Gauss'
./section3.tex:377: Gauss'  ==> Gauss'
./section3.tex:385: Gauss'  ==> Gauss'
```

larsoner commented:
> Is the dictionary case insensitive?

Yes at this point it just maintains the case of whatever the original spelling was IIRC. At some point we might add better case handling, though.

You have an error:

> AssertionError: currently corrections must end with trailing "," (if  multiple corrections are available) or have "disabled" in the comment

banesullivan commented:
Okay I made this so that it only corrects misspellings Gauss and suggests a correction with a capital G. Thus "gauss" would be ignored by codespell because it isn't case-sensitive

banesullivan commented:
and `becausw->because` which is a really common one for me

banesullivan commented:
I've been writing a lot of lecture slides lately and I added all the mispellings I made that `codespell` didn't recognize.

I also added `,easure->measure`. Will that mess up the script because it parses the dictionary using commas?

banesullivan commented:
Welp, travis answered that question

larsoner commented:
Thanks @banesullivan

lurch commented:
Not that it really matters, but FYI https://en.wikipedia.org/wiki/Gauss_(unit)#Name,_symbol,_and_metric_prefixes says

> When the unit is spelled out, it is written in lowercase ("gauss"), unless it begins a sentence.
